pkg/controller/system/system_controller.go

```diff
@@ -874,9 +874,8 @@ func (r *ReconcileSystem) ReconcileCertificates(client *gophercloud.ServiceClien
 				return err
 			}
 
-			msg := fmt.Sprintf("waiting for %q certificate %q to be created", c.Type, c.Secret)
-			r.WarningEvent(instance, common.ResourceDependency, msg)
-			return common.NewMissingKubernetesResource(msg)
+			log.Info(fmt.Sprintf("skipping %q certificate %q from system", c.Type, c.Secret))
```

Comment:
I think we need to state why this dependency is no longer being enforced.  In general it means we can now have a misconfigured Certificate entry pointing to a non-existent Secret.  Likely this needs to be more than just an Info log since it could also be a misconfiguration.  So keep the event, and log a warning.